components/View/index.jsx

```diff
@@ -56,6 +58,8 @@ class View extends Component {
   constructor(props) {
     super(props);
 
+    // Store the active pathname at instantiation
```

Comment:
`// Store the active pathname on instantiation.`

Reply:
done